classes/main.py

```diff
@@ -39,6 +39,9 @@
     #background
     display_surface.blit(bg_surf,(0,0))
 
+    #updates
+    player_group.update()
```

Comment:
I think it is a better idea to update after rendering the player.

Reply:
I believe it would work fine now as we have minimal tasks. However,  my sole purpose in adding it before rendering a player is, to call update function capture player movements to render it to the UI.